dealership/contracts.py

```diff
@@ -1,12 +1,61 @@
 class Contract(object):
-    pass
+
+    def __init__(self, vehicle, customer):
+        self.vehicle = vehicle
+        self.customer = customer
+
+    def get_discount(self):
+        if self.customer.is_employee():
+            discount = .1
+        else:
+            discount = 0
+        return discount
 
 
 class BuyContract(Contract):
+
+    interest = {
```

Comment:
I'd rather put it as attributes in Car, Motorcycle and Truck classes. This way it's easier to find and you have all multipliers in one place.

Reply:
Good idea. I don't know why I didn't think of that. I spent some time researching how to get the name of an object as a string, when it really could have been a lot simpler.